packages/react/src/primitives/Select/__tests__/Select.test.tsx

```diff
@@ -101,14 +127,9 @@ describe('Select primitive test suite', () => {
     expect(optionThree.selected).toBeTruthy();
   });
 
-  it('should not be able to select an option if disabled', async () => {
+  it('should be able to select an option as an uncontrolled component', async () => {
     render(
-      <Select
-        testId={testId}
-        placeholder={placeholder}
-        defaultValue=""
-        isDisabled
-      >
+      <Select testId={testId} placeholder={placeholder} defaultValue="">
```

Comment:
For the uncontrolled component we should confirm that passing `defaultValue="2"` actually sets that value instead of the placeholder. Should we add a test variation that sets the defaultValue to 2?

Reply:
I don't think it is necessary? I think the `defaultValue` is a behavior of React select for uncontrolled component to initialize. At least, I don't think `defaultValue` will work when `value` present at the same time.